diff --git a/packages/contracts-sdk/src/Cw721Roles.client.ts b/packages/contracts-sdk/src/Cw721Roles.client.ts
index 84c306e..216fb37 100644
--- a/packages/contracts-sdk/src/Cw721Roles.client.ts
+++ b/packages/contracts-sdk/src/Cw721Roles.client.ts
@@ -84,7 +84,7 @@ export interface Cw721RolesReadOnlyInterface {
     startAfter?: string;
   }) => Promise<TokensResponse>;
   minter: () => Promise<MinterResponse>;
-  extension: ({
+  getExtension: ({
     msg
   }: {
     msg: QueryExt;
@@ -110,7 +110,7 @@ export class Cw721RolesQueryClient implements Cw721RolesReadOnlyInterface {
     this.tokens = this.tokens.bind(this);
     this.allTokens = this.allTokens.bind(this);
     this.minter = this.minter.bind(this);
-    this.extension = this.extension.bind(this);
+    this.getExtension = this.getExtension.bind(this);
     this.ownership = this.ownership.bind(this);
   }
 
@@ -267,7 +267,7 @@ export class Cw721RolesQueryClient implements Cw721RolesReadOnlyInterface {
       minter: {}
     });
   };
-  extension = async ({
+  getExtension = async ({
     msg
   }: {
     msg: QueryExt;
